app/views/notes/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class='container'>
```

Comment:
make it double quoted as most classes are double quoted in this file

Reply:
all converted to single quotes